python/src/StationaryFunctionalCovarianceModel_doc.i.in

```diff
@@ -0,0 +1,49 @@
+%feature("docstring") OT::StationaryFunctionalCovarianceModel
+"Functional covariance function.
+
+Allows to define an 1-d stationary covariance model from a custom correlation function.
+
+Parameters
+----------
+scale : sequence of floats
+    Scale coefficient :math:`\vect{\theta}\in \Rset^n`.
+    The size of :math:`\vect{\theta}` is the input dimension.
+    Must be of size equal to 1.
+amplitude : sequence of positive floats
+    Amplitude  :math:`\vect{\sigma}\in \Rset^d`.
+    Must be of size equal to 1.
+    By default, equal to :math:`[1]`.
+rho : :class:`~openturns.Function`
+    Must have 1-d 1-d output
+    Correlation function :math:`\rho(\tau)`
+    No check is made that this is actually a correlation function
+    so the resulting discretized matrix can be non SPD if you're not careful enough
+
+Notes
+-----
+The functional covariance function is a stationary covariance function dimension :math:`d=1`.
+
+We consider the scalar stochastic process :math:`X: \Omega \times\cD \mapsto \Rset`, where :math:`\omega \in \Omega` is an event, :math:`\cD` is a domain of :math:`\Rset^n`.
+
+The functional covariance function is defined by:
+
+.. math::
+
+    C(\vect{s}, \vect{t}) = \sigma^2 \rho(\vect{s} - \vect{t}), \quad \forall (\vect{s}, \vect{t}) \in \cD
+
+with :math:`\rho(\tau)` the provided correlation function.
+
+See Also
+--------
+CovarianceModel
+
+Examples
+--------
+Create a covariance model defined by a damped cosine correlation function:
```

Comment:
Isn't that an existing ``CovarianceModel`` in the library? Wouldn't it be better to choose something that is unavailable?